.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     if: ${{ always() }}
     runs-on: ubuntu-latest
     container:
-      image: docker.io/zmkfirmware/zmk-build-arm:3.5
+      image: docker.io/zmkfirmware/zmk-build-arm:3.5-branch
```

Comment:
This is temporary while I get our `zmk-docker` images for the `3.5` tag to actually publish.